### PR TITLE
Add page on OxVPS

### PR DESCRIPTION
**Pull request by tlestang**
In `projects/`

### CODE REVIEW

**Comment by fcooper8472** [comment]
Thanks!  I'll add some to the branch.

Do you have those screenshots in png format by any chance?

File: content/projects/2019_oxvps.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+     
+## OxRSE involvement
+
+{{< figure src="/images/projects/OxVPS_screenshots.jpg" caption="Screenshots of the mobile application developed by the OxRSE team for the OxVPS project.">}}
+
+OxRSE was involved from the very early stages of the OxVPS project. Fergus Cooper, Mihaela Duta and Thibault Lestang provided both the software expertise and development work to turn the paper-based assessment into a full-blown mobile app, usable by clinicians with very limited time and suitable for stroke patients with impairments such as neglecting objects on one side or difficulties recognising objects.
+     
+We also worked closely with Dr Kathleen Van Cleef and Federica Guazzo to implement an automatic scoring system, as well as the gathering of test data necessary to validate the application-based assessment against trusted paper-based visual perception tests.
```
Comment (iamleeg) [inline content/projects/2019_oxvps.md:19]: typo: should be Vancleef (Van Cleef is a jeweller I think)

**Comment by tlestang** [comment]
@fcooper8472 
> Thanks! I'll add some to the branch.
> 
> Do you have those screenshots in png format by any chance?

No, but I can make them PNG images if you want. Or I can export the whole image as a png?

File: content/projects/2019_oxvps.md (hunk 1 above)
Comment (tlestang) [reply content/projects/2019_oxvps.md:19]: Oops

**Comment by fcooper8472** [comment]
> No, but I can make them PNG images if you want. Or I can export the whole image as a png?

if it would be easy to get the original screenshots as png, and keep them as png for the resulting image, they will end up looking sharper.  But don't worry if it's not straightforward - it's not a big deal!

**Comment by tlestang** [comment]
Thanks @fcooper8472 , are you happy for it to be merged now?

**Comment by fcooper8472** [comment]
Yep, happy for this to go in!

**Pull request merged**